packages/app/src/vite-env.d.ts

```diff
@@ -1 +1,9 @@
 /// <reference types="vite/client" />
+
+interface ImportMetaEnv {
```

Comment:
What do the additions to this file do?

Reply:
provides type definitions for `import.meta.env`, but I see this project is using `process.env` only

I will remove then and we can add later if we want to change to `import.meta.env`